Makefile

```diff
@@ -4,7 +4,7 @@ IMAGE_PREFIX                := $(REGISTRY)
 REPO_ROOT                   := $(shell dirname $(realpath $(lastword $(MAKEFILE_LIST))))
 HACK_DIR                    := $(REPO_ROOT)/hack
 HOSTNAME                    := $(shell hostname)
-LD_FLAGS                    := "-w -X github.com/gardener/gardener-extensions/pkg/version.Version=$(IMAGE_TAG)"
+LD_FLAGS                    := "-w -X github.com/metal-stack/gardener-extension-provider-metal/pkg/version.Version=$(IMAGE_TAG)"
```

Comment:
I think this must be: 
```
github.com/metal-stack/v.Version=$(IMAGE_TAG)
```
because the given package does not exist.

Reply:
I just took this from here https://github.com/gardener/gardener-extension-provider-aws/blob/master/Makefile#L23. Doesn't mean I understand it.

Reply:
This mechanism requires that this Variable is present:
```
github.com/metal-stack/gardener-extension-provider-metal/pkg/version.Version
```
which is not the case as far as i can see, i can check.

Reply:
Yeah, for some reason this package is not present in the original repositories either...